test/integration/controllers/category.spec.js

```diff
@@ -38,14 +93,19 @@ describe('Category controller', () => {
 
     accessToken = await testUserAuthentication(app)
 
-    testCategory = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send(categoryBody)
+    for (let i = 0; i < categoryBody.length; i++) {
+      const category = categoryBody[i]
+      const subject = subjectBody[i]
+
+      testCategory = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send(category)
```

Comment:
why do we need to create categories manually if checkCategoryExistence() already does it?

Reply:
Fixed